crds/operators.coreos.com_clusterserviceversions.yaml

```diff
@@ -8988,6 +8992,27 @@ spec:
                         type: string
                       name:
                         type: string
+                release:
+                  description: |-
+                    release specifies the packaging version of the operator, defaulting to empty
+                    release is optional
+
+                    A ClusterServiceVersion's release field is used to provide a secondary ordering
+                    for operators that share the same semantic version. This is useful for
+                    operators that need to make changes to the CSV which don't affect their functionality,
+                    for example, to update their base image, or to fix a typo in their description.
```

Comment:
I would argue that a base image change should require a bump of the version's patch number, not a bump of a release number. I know in the past the substitutesFor logic was specifically built to handle rebuild scenarios. But I think from a user trust standpoint, base image changes are happening generally in order to make security fixes, and to me, that falls squarely in patch territory, not re-release territory.